fuel-vm/src/interpreter/internal.rs

```diff
@@ -89,8 +89,8 @@ pub(crate) fn update_memory_output<Tx: ExecutableTransaction>(
         .ok_or(PanicReason::OutputNotFound)?;
     let mut mem = memory.write_noownerchecks(range.start, range.len())?;
     let output = tx
-        .outputs_mut()
-        .get_mut(idx)
+        .outputs()
+        .get(idx)
         .expect("Invalid output index; checked above");
     output
```

Comment:
I remember why it was mut. Because `encode` before require mutable reference...) After refactoring of encoding and decoding now it is not an issue